tests/flow/test_ts_delete.py

```diff
@@ -120,3 +120,99 @@ def test_bad_del(self):
         self.assertTrue(r.execute_command("SET", "BAD_X", "NOT_TS"))
         with pytest.raises(redis.ResponseError) as excinfo:
             r.execute_command("TS.DEL", "BAD_X", 100, 200)
+
+def test_del_retention_with_rules(self):
```

Comment:
@OfirMos by default always test with compressed because that is the common case. but if possible make the test parameterized and test both of them.